website/client/router.js

```diff
@@ -81,7 +80,6 @@ export default new VueRouter({
       ],
     },
     { name: 'challenges', path: 'challenges', component: Page },
-    { name: 'party', path: 'party', component: Page },
     {
       path: '/user',
       component: ParentPage,
```

Comment:
outside the scope of this PR but this component should be changed with one that contains the submenu, will do when we implement the users views